internal/controller/nodeagent.go

```diff
@@ -253,7 +253,9 @@ func (r *DataProtectionApplicationReconciler) ReconcileNodeAgentDaemonset(log lo
 			veleroAffinityStruct := make([]*kube.LoadAffinity, len(dpa.Spec.Configuration.NodeAgent.NodeAgentConfigMapSettings.LoadAffinityConfig))
 
 			for i, aff := range dpa.Spec.Configuration.NodeAgent.NodeAgentConfigMapSettings.LoadAffinityConfig {
-				veleroAffinityStruct[i] = (*kube.LoadAffinity)(aff)
+				veleroAffinityStruct[i] = &kube.LoadAffinity{
+					NodeSelector: aff.NodeSelector,
+				}
```

Comment:
Add StorageClass to dpa.LoadAffinity type
velero added StorageClass to affinity so follow up to add storageclass to our [types](https://github.com/kaovilai/oadp-operator/blob/dde9fe2da2f6e689d25e513f7d9b3c2368efe4aa/api/v1alpha1/dataprotectionapplication_types.go#L399-L403)
from velero pkg/util/kube/pod.go
```go
type LoadAffinity struct {
	// NodeSelector specifies the label selector to match nodes
	NodeSelector metav1.LabelSelector `json:"nodeSelector"`

	// StorageClass specifies the VGDPs the LoadAffinity applied to. If the StorageClass doesn't have value, it applies to all. If not, it applies to only the VGDPs that use this StorageClass.
	StorageClass string `json:"storageClass"`
}
```